test/manipulation.test.js

```diff
@@ -1097,6 +1098,53 @@ describe('manipulation', function() {
     });
   }
 
+  var hasReplaceById = !!getSchema().connector.replaceById;
+  bdd.describeIf(hasReplaceById, 'replaceOrCreate when forceId is true', function() {
+    var Post, foundId;
+    var ds = getSchema();
+    before(function(done) {
+      Post = ds.define('Post', {
+        title: {type: String, length: 255, index: true},
+        content: {type: String},
+        comments: [String],
```

Comment:
I think the `content` field is not needed either, and the `title` field dos not need `length` nor `index` config.

Please keep things as simple as possible, to prevent future confusion about whether this particular model configuration is required for the tests (and why?).